README.md

```diff
@@ -34,14 +34,24 @@ Edit this document to include your answers after each question. Make sure to lea
 
 1. What is the difference between an adaptive website and a fully responsive website?
 
+Adaptive sites have media queries to ensure that design will switch to a friendlier version at certain markers (500px, 800px, etc.). Whereas fully responsive design grows and shrinks with any screen size, not just when the marker is hit. Responsive sites will accomodate any width or orientation by contracting and expanding precisely with the page.
+
 2. Describe what it means to be mobile first vs desktop first.
 
+This simply means whether the design was originally made for a mobile device, then scaled up with media queries to be used for desktop, or if it was originally designed for desktop, then scaled down for mobile devices. 
+
 3. What does `font-size: 62.5%` in the `html` tag do for us when using `rem` units?
 
+Due to the browser's default font size of 16px, this percentage gives us a font size of 10px in the root html element. Having such an easy number to work with in the root element means that scaling other things on the page is simple by way of root-element units, or rem. (Ex: 17 px = 1.7rem;)
+
 4. How would you describe preprocessing to someone new to CSS?
 
+Preprocessing is like having another type of css that's easier to work with when you style your pages. When you make changes in the easy css, the preprocessor then does the heavy lifting of converting the easy css into real css for the browser to read.
```

Comment:
I love this definition! Someone that has no coding experience could read this and have a basic understanding of how Preprocessors work.